1906-10-05.xml

```diff
@@ -0,0 +1,7096 @@
+                </div>
+                <div type="advert" xml:id="deg-ad-cah04">
+                    <head>CARLTON HOTEL</head>
+                    <p>Bulkeley (near Alexandria) half way to San Stefano</p>
+                    <p>Ramleh's Fashionable Hotel.</p>
+                    <p>Full Pension P.T. 50 a day with Monthly Terms</p>
+                    <p>Visitors from Cairo alight at Sidi Gaber.</p>
+                    <p>Proprietor, C. AQUILINA, (Late of Thos Cook &amp; Sons)</p>
+                    <p><measure>36-11-</measure></p>
+                </div>
+                <div type="advert" xml:id="deg-ad-haa01">
+                    <head>Hamburg &amp; Anglo-American Nile Company.</head>
+                    <p>River Transport of Goods between Alexandria and Cairo.</p>
+                    <p>Three Sailings a-week.</p>
+                    <p>Agents at Alexandria</p>
+                    <p>ALEXANDRIA BONDED WAREHOUSE CO. LTD.</p>
+                    <p><cb/></p>
+                </div>
+                <div type="section">
+                    <head>THE ISIS DISASTER.</head>
+                    <p>MAILS ON SUNDAY.</p>
+                    <p>We are informed by the bad office of the Postal Administration that the
+                        Brindisi mail</p>
+                    <p>will arrive at Port Said at six o'clock tomorrow evening and that the
+                        Alexandria distribution</p>
+                    <p>will commence at 8 a.m. on Sunday. It will be observed from the Reuter's
+                        telegram which we append that the Isis has been sufficiently repaired to be
+                        able to continue her voyage. Doubtless she will be accompanied by the
+                        Persis, which as we reported yesterday, went to her help on receipt of the
+                        bows at Port Baid, so that in the event of further break down every
+                        assistance will be at hand.</p>
+                    <p>Among the passengers who, according to Reuter's despatch of the day before
+                        yesterday,</p>
+                    <p>ran so great a risk of serious injury; were Mr. H. Ampinall, the popular
+                        secretary to the Torf</p>
+                    <p>Clab, Cairo, and Mr. George Walker, of the firm of Messrs. Walker and
+                        Meimaracbi. </p>
+                    <p>The weekly Brindisi mail will be made up at the G. P. O., Alexandria, on
+                        Sunday at</p>
+                    <p>3.10 p.m. for ordinary correspondence. Registered letters must be posted one
+                        hour earlier.</p>
+                    <p>ZANTE, October 4. </p>
+                    <p>The Isis has left with passengers and mails for Port Ssid, where she is
+                        expected to arrive on Saturday. (Reuter)</p>
+                </div>
+                <div type="item">
```

Comment:
this is `<div type="section" feature="letters>`. The break each letter into its own item, with head and byline and dateline.